libs/api/database/src/lib/content-library/schemas/index.ts

```diff
@@ -0,0 +1,62 @@
+import { ContentLibrarySchema } from './content-library.schema';
+import { BookshelfDocument, BookshelfSchema } from './bookshelf.schema';
+import { ShelfItemSchema } from './shelf-item.schema';
+import * as MongooseAutopopulate from 'mongoose-autopopulate';
+import * as MongoosePaginate from 'mongoose-paginate-v2';
+import { HookNextFunction } from 'mongoose';
+import * as sanitizeHtml from 'sanitize-html';
+import { sanitizeOptions } from '@dragonfish/shared/models/util';
+
+//#region ---EXPORTS---
+
+export { ContentLibraryDocument, ContentLibrarySchema } from './content-library.schema';
+export { BookshelfDocument, BookshelfSchema } from './bookshelf.schema';
+export { ShelfItemDocument, ShelfItemSchema } from './shelf-item.schema';
+
+//#endregion
+
+//#region ---SCHEMA FACTORIES---
+
+/**
+ * Sets up the content library collection.
+ */
+export async function setupContentLibraryCollection() {
+    const schema = ContentLibrarySchema;
+
+    schema.plugin(MongooseAutopopulate);
+    schema.plugin(MongoosePaginate);
+
+    return schema;
+}
+
+/**
+ * Sets up the bookshelves collection.
+ */
+export async function setupBookshelvesCollection() {
```

Comment:
Should this be in an index file?

Reply:
that's how it goes for all the updated database modules; it helps to keep clutter out of the module file itself so it's easier to read overall